src/main/drivers/osd.h

```diff
@@ -48,6 +48,7 @@ typedef enum {
     VIDEO_SYSTEM_NTSC,
     VIDEO_SYSTEM_HDZERO,
     VIDEO_SYSTEM_DJIWTF,
+    VIDEO_SYSTEM_AVATAR,
```

Comment:
Ideally, when changing an enum you should add new values at the end, otherwise it can break backwards compatibility. (BFCOMPAT went from 5 to 6 in this change)  Not an issue here, as both entries are new in this major release and the configurator settings match.

Just make sure it matches the values in the configurator as well.

Reply:
Yes, I agree.. The reason I added AVATAR before BFCOMPAT is because where DJI reached out to Pawel now. I figured that there could be a time in the future when BFCOMPAT could be replaced with the actual DJI O3 implementation. ...if it happens. I figured it hasn't been released yet, so no harm.

Everything matches in Configurator.